ios/brave-ios/App/Client.xcodeproj/project.pbxproj

```diff
@@ -995,7 +995,7 @@
 			attributes = {
 				DefaultBuildSystemTypeForWorkspace = Latest;
 				LastSwiftMigration = 0700;
-				LastSwiftUpdateCheck = 1500;
+				LastSwiftUpdateCheck = 1530;
```

Comment:
question: Why do we need the update?

Reply:
Happened automatically, something Xcode just does on its own I guess?